data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -1,7 +1,19 @@
-WITH t_hypothesis_annotation_with_doi AS (
+WITH t_hypothesis_annotation_with_cleaned_uri AS (
   SELECT
-    * EXCEPT (id, created),
-    CASE 
+    * EXCEPT (id, created, uri),
+    annotation.id AS hypothesis_id,
+    annotation.created AS annotation_created_timestamp,
+    DATE(annotation.created) AS annotation_created_date,
+    REGEXP_REPLACE(uri, r'/$', '') AS uri
```

Comment:
Just a FYI, [RTRIM](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#rtrim) can also be used to remove trailing characters (not just space), e.g.:

```sql
RTRIM(uri, '/')
```